doc/mkdocs/mkdocs.yml

```diff
@@ -31,6 +31,7 @@ theme:
 nav:
   - Home:
     - index.md
+    - "API Documentation": api/basic_json/index.md
```

Comment:
I get why you're doing this, but I think it's confusing to have the menu item twice - especially since the behavior of the other pages is different.

Reply:
I did not plan to keep it there. I noticed the other day, that I'm zoomed in to 150% on json.nlohmann.me which removes the menu bar. I was missing a direct link to the API documentation all this time. :-)